pages/doc/wavefront_prometheus.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Using Prometheus Queries in Wavefront
+keywords: query language
+sidebar: doc_sidebar
+published: true
+permalink: wavefront_prometheus.html
+summary: Run Prometheus from the Wavefront query line
+---
+
+Starting with release 2020.26, Wavefront includes support for Prometheus queries.
+* Run Prometheus queries directly from the Wavefront query line. The corresponding chart shows the information you'd expect.
+* Wavefront query language includes new functions and operators that allow you to "translate" any Prometheus query to a ts() query and use it with the rich set of existing ts() query language functions.
+
+## How to Run Prometheus Queries
+
+1. Create or edit a chart.
+2. Start typing your Prometheus query .
+3. Make changes to the visualization.
+  * See [Create and Customize Charts](ui_charts.html) for an intro.
+  * See [Chart Reference](ui_chart_reference) for details.
+
+![Prometheus query](images/prometheus_sample.png)
+
+## How Prometheus Queries Can Differ in Wavefront.
+
+Wavefront supports all Prometheus functions out of the box. There are a small number of differences.
```

Comment:
One thing to note is that you're using the word functions here but PromQL defines functions in a different way than we do.  It has functions + operators.  See the slides of this tech talk: https://onevmw-my.sharepoint.com/:p:/g/personal/songhao_vmware_com/EfN7B6RNv_NFuzG2jMWymRwBxHcwWWAdASIukg8Qgp55Dg

But in short i think since this will be read by PromQL users we should use their nomenclature and talk about functions as well as operators at a minimum.  You can also make a call if you want to talk about functions not supported or operators not supported separately but i'm fine with either.

Reply:
Changed to "functions and operators".